README.md

```diff
@@ -0,0 +1,210 @@
+
+3. Edit the configuration file (data_dir/your_config.yaml) to tweak values present
+   in the `gce:` session of that file. One of the values you might want to
+   tweak is the scylla yum repository used to install scylla on the CentOS 7 image.
+
+With that said and done, you can run your test using the command line::
+```bash
+hydra run-test longevity_test.LongevityTest.test_custom_time --backend gce --config test-cases/scylla-lmr.yaml
+```
+
+### Azure
+
+You can run your test using the command line:
+```bash
+hydra run-test longevity_test.LongevityTest.test_custom_time --backend azure --config test-cases/PR-provision-test.yaml
+```
+
+## Types of Tests
+### [Artifact tests](./docs/artifacts_test.md)
+### Longevity Tests (TODO: write explanation for them)
+### Upgrade Tests (TODO: write explanation for them)
+### Performance Tests (TODO: write explanation for them)
+### Features Tests (TODO: write explanation for them)
+### Manager Tests (TODO: write explanation for them)
+### [K8S Functional Tests](./docs/k8s-functional-test.md)
+### [Microbenchmarking Tests](./docs/microbenchmarking.md)
+
+
+## Test operations
+
+On a high level overview, the test operations are:
+
+### Setup
+
+1) Instantiate a Cluster DB, with the specified number of nodes (the number
+   of nodes can be specified through the config file, or the test writer can
+   set a specific number depending on the test needs).
+
+2) Instantiate a set of loader nodes. They will be the ones to initiate
+   cassandra stress, and possibly other database stress inducing activities.
+
+3) Instantiate a set of monitoring nodes. They will run prometheus [3], to
+   store metrics information about the database cluster, and also grafana [4],
+   to let the user see real time dashboards of said metrics while the test is
+   running. This is very useful in case you want to run the test suite and keep
+   watching the behavior of each node.
+
+4) Wait until the loaders are ready (SSH up and cassandra-stress is present)
+
+5) Wait until the DB nodes are ready (SSH up and DB services are up, port 9042
+   occupied)
+
+6) Wait until the monitoring nodes are ready. If you are following the job log,
+   you will see a message with the address you can point your browser to while
+   the test is executing ::
+
+    02:09:37 INFO | Node lmr-scylla-monitor-node-235cdfb0-1 [54.86.66.156 | 172.30.0.105] (seed: None): Grafana Web UI: http://54.86.66.156:3000
+
+### Actual test
+
+1) Loader nodes execute cassandra stress on the DB cluster (optional)
+
+2) If configured, a Nemesis class, will execute periodically, introducing some
+   disruption activity to the cluster (stop/start a node, destroy data, kill
+   scylla processes on a node). the nemesis starts after an interval, to give
+   cassandra-stress on step 1 to stabilize
+
+Keep in mind that the suite libraries are flexible, and will allow you to
+set scenarios that differ from this base one.
+
+## Directory Structure of the project
```

Comment:
this part of the structure tree is so not up to date, I'm wondering if there a point explaining all the structure ? maybe in contrib section ?

Reply:
contrib section sounds good

Reply:
moved it